app/views/shared/_navbar.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <div class="container-fluid">
-    <%= link_to "#", class: "navbar-brand" do %>
-      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
+    <%= link_to tours_path, class: "navbar-brand" do %>
```

Comment:
Can you make it go to the home page instead of the index of tours?

Reply:
You are right, I will change

Reply:
I have changed it